benchmark/src/main/scala/org/virtuslab/akkasaferserializer/benchmark/Main.scala

```diff
@@ -0,0 +1,48 @@
+package org.virtuslab.akkasaferserializer.benchmark
+
+object JavaSerialization {
+  def main(args: Array[String]): Unit = {
+    Benchmark.benchmark("application-java.conf", includeCommon = false, "json/java.txt")
+    System.exit(0)
+  }
+
+}
+
+object JacksonJson {
+  def main(args: Array[String]): Unit = {
+    Benchmark.benchmark("application-jackson.conf", includeCommon = false, "json/jackson.txt", assertEquality = false)
+    System.exit(0)
+  }
+}
+
+object BorerJson {
+  def main(args: Array[String]): Unit = {
+    Benchmark.benchmark("application-borer-json.conf", includeCommon = true, "json/borer.txt")
+    System.exit(0)
+  }
+}
+
+object JacksonCbor {
+  def main(args: Array[String]): Unit = {
+    Benchmark.benchmark(
+      "application-jackson-cbor.conf",
+      includeCommon = false,
+      "cbor/jackson.txt",
+      assertEquality = false)
+    System.exit(0)
+  }
+}
+
+object BorerCbor {
+  def main(args: Array[String]): Unit = {
+    Benchmark.benchmark("application-borer.conf", includeCommon = true, "cbor/borer.txt")
```

Comment:
Whoops getting:

```
sbt:akka-serialization-helper> benchmark/run
[warn] multiple main classes detected: run 'show discoveredMainClasses' to see the list

Multiple main classes detected. Select one to run:
 [1] org.virtuslab.akkasaferserializer.benchmark.BorerCbor
 [2] org.virtuslab.akkasaferserializer.benchmark.BorerJson
 [3] org.virtuslab.akkasaferserializer.benchmark.JacksonCbor
 [4] org.virtuslab.akkasaferserializer.benchmark.JacksonJson
 [5] org.virtuslab.akkasaferserializer.benchmark.JavaSerialization
 [6] org.virtuslab.akkasaferserializer.benchmark.Kryo

Enter number: 1
[info] running org.virtuslab.akkasaferserializer.benchmark.BorerCbor 
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
[error] (run-main-0) java.io.FileNotFoundException: cbor/borer.txt (No such file or directory)
[error] java.io.FileNotFoundException: cbor/borer.txt (No such file or directory)
[error] 	at java.base/java.io.FileOutputStream.open0(Native Method)
[error] 	at java.base/java.io.FileOutputStream.open(FileOutputStream.java:298)
[error] 	at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:237)
[error] 	at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:126)
[error] 	at java.base/java.io.PrintWriter.<init>(PrintWriter.java:200)
[error] 	at org.virtuslab.akkasaferserializer.benchmark.Benchmark$$anon$1.<init>(Benchmark.scala:20)
[error] 	at org.virtuslab.akkasaferserializer.benchmark.Benchmark$.benchmark(Benchmark.scala:20)
[error] 	at org.virtuslab.akkasaferserializer.benchmark.BorerCbor$.main(Main.scala:38)
[error] 	at org.virtuslab.akkasaferserializer.benchmark.BorerCbor.main(Main.scala)
[error] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[error] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[error] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error] 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[error] stack trace is suppressed; run last benchmark / Compile / bgRun for the full output
```

Looks like there's a silent assumption that `cbor/` directory already exists